src/services/item/plugins/action/base-analytics.ts

```diff
@@ -68,13 +62,17 @@ export class BaseAnalytics {
     // TODO: all other schemas
 
     // validate and remove additional properties from member
-    const ajv = new Ajv({ removeAdditional: 'all' });
+    const ajv = new Ajv({ removeAdditional: 'all' }).addFormat(
+      'username',
```

Comment:
Maybe add a comment why we need it. It might mean that we should add all the other definitions? But for now if it solves your problem it's alright.

Reply:
Actually this did not solve the issue of the un-recognized format since the error was not originating from here.
I have removed this change and instead found a fix for the format.